nano/node/websocket.cpp

```diff
@@ -42,11 +81,12 @@ void nano::websocket::session::close ()
 	ws.close (reason, ec_ignore);
 }
 
-void nano::websocket::session::write (nano::websocket::message message_a)
+void nano::websocket::session::write (nano::websocket::message message_a, nano::node & node_a)
```

Comment:
No need to pass the node here, it's available through `ws_listener.get_node ()` everywhere in the session.